airflow/api/client/json_client.py

```diff
@@ -92,3 +92,13 @@ def delete_pool(self, name):
         url = urljoin(self._api_base_url, endpoint)
         pool = self._request(url, method='DELETE')
         return pool['pool'], pool['slots'], pool['description']
+
+   def cancel_dag_run(dag_id, dagrun_run_id): 
```

Comment:
mypy?

Reply:
airflow codebase doesnt do mypy

Reply:
:( i didnt realize we wanted to merge this upstream. seems fine then